docs/cli.md

```diff
@@ -93,6 +94,29 @@ In the case where your application resides in a directory other than your reposi
 folder, you can specify that directory using the `--context-dir` parameter. The
 specified directory will be used as your application root folder.
 
+#### Injecting directories to build
+
+In case you want to inject files that should be available only during the
+build time (iow. when the `assemble` script is invoked), you can specify the
+directories from where the files will be copied into the container that runs
+this script. To do so you can invoke S2I as following:
```

Comment:
"If you want to inject files that should only be available during the build (ie when the `assemble` script is invoked), you can specify the directories from which the files will be copied into the container that runs the assemble script.  To do so you can invoke S2I as follows:"